app/lib/methods/helpers/openLink.ts

```diff
@@ -37,9 +38,9 @@ const appSchemeURL = (url: string, browser: string): string => {
 };
 
 const openLink = async (url: string, theme: TSupportedThemes = 'light'): Promise<void> => {
+	url = addProtocol(url);
```

Comment:
Any reason not to add this to `try/catch`?

Reply:
Because at line 58 (https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/fb47f8221fd7bfb6bd9703873d255c435a52a9d8/app/lib/methods/helpers/openLink.ts#L58) at the catch, it's expected to use the URL